src/shared/install.c

```diff
@@ -3625,6 +3625,14 @@ Hashmap* unit_file_list_free(Hashmap *h) {
 
 DEFINE_TRIVIAL_CLEANUP_FUNC(UnitFileList*, unit_file_list_free_one);
 
+DEFINE_HASH_OPS_WITH_VALUE_DESTRUCTOR(
+        unit_file_list_hash_ops,
```

Comment:
I think this should be called `unit_file_list_hash_ops_free`, because it frees the value.

Reply:
Indeed. Changed.